src/js/_enqueues/wp/embed.js

```diff
@@ -11,18 +11,15 @@
 (function ( window, document ) {
 	'use strict';
 
-	var supportedBrowser = false,
-		loaded = false;
-
-		if ( document.querySelector ) {
-			if ( window.addEventListener ) {
-				supportedBrowser = true;
-			}
-		}
+	// Abort for ancient browsers.
```

Comment:
IIRC this whole file does not use `//` comments, but only `/* */` comments because the script will be inlined in `get_post_embed_html()`. So the former type of comment would break the whole script.

This is for the case when someone copies the embed code from the embed block and tries to embed it on a non-WP site.

So keeping the `/* */` comment style is important.

Reply:
Thank you. Even though minification strips out comments?

Reply:
Not when using `SCRIPT_DEBUG`

Reply:
Rectified in 7d18b4b66f